lib/single-new-eks-opensource-observability-pattern/neuron/neuron-device-plugin-addon.ts

```diff
@@ -0,0 +1,44 @@
+import { KubernetesManifest, ServiceAccount } from "aws-cdk-lib/aws-eks";
```

Comment:
You should move this to blueprints repo. I see an active PR - https://github.com/aws-quickstart/cdk-eks-blueprints/pull/777 for this. Lets get this from blueprints vs using it here. Having in blueprints make this available to all.